tests/MongoDB.Driver.Core.Tests/MongoDB.Driver.Core.Tests.csproj

```diff
@@ -39,7 +39,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="DnsClient" Version="1.2.0" />
+    <PackageReference Include="DnsClient" Version="1.3.0" />
```

Comment:
I wonder why we have a dependency on the DnsClient package in this project?

Reply:
done